src/hotspot/share/memory/virtualspace.cpp

```diff
@@ -192,8 +190,8 @@ void ReservedSpace::initialize(size_t size, size_t alignment, bool large,
              "Large pages returned a non-aligned address, base: "
              PTR_FORMAT " alignment: " SIZE_FORMAT_HEX,
              p2i(base), alignment);
-      _special = true;
     } else {
+      _special = false;
```

Comment:
Changes (special->_special) are Cleanup, please do them in a separate RFE if needed.